backend/src/chats/message/message.controller.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from '@nestjs/common';
 import { MessageService } from './message.service';
 import { CreateMessageDto } from './dto/create-message.dto';
-import { UpdateMessageDto } from './dto/update-message.dto';
+// import { UpdateMessageDto } from './dto/update-message.dto';
```

Comment:
is this comment still needed?

and in the top most import the "patch" import is unused at the moment (but I cannot comment specifically on that, because github does not allow me.....)